dev-packages/test-utils/src/event-proxy-server.ts

```diff
@@ -247,49 +282,54 @@ export async function waitForPlainRequest(
 /** Wait for a request to be sent. */
 export async function waitForRequest(
   proxyServerName: string,
+  timestamp: number,
```

Comment:
m: Can we make this optional, as third argument, and just default to `Date.now()`? I guess we'll want this like this basically always? 😅  (same for the other methods)

Reply:
yeah made it optional